package.json

```diff
@@ -1,33 +1,70 @@
 {
-  "name": "daskeyboard-Applet-Github-Action",
+  "name": "daskeyboard-aplet--action-status-for-github",
+  "displayName": "Action Status for GitHub",
   "version": "1.0.0",
-  "description": "Tracking the progress of GitHub Actions within a repository",
-  "main": "index.js",
-  "repository": "https://github.com/daskeyboard/daskeyboard-Github-Actions-Applet",
-  "author": "Soulaymane Kebli",
+  "description": "Track the status of GitHub Actions within a repository",
+  "longDescription": "Stay updated on the progress of your GitHub Actions workflows with a single key on your Das Keyboard.",
+  "isSingleton": false,
+  "icon": "assets/icon.png",
+  "image": "assets/image-missing.png",
+  "publisher": "SoulaymaneK",
+  "authorName": "Soulaymane Kebli",
+  "licenseUrl": "https://example.com/license",
+  "authorUrl": "https://github.com/SoulaymaneK",
+  "issuesUrl": "https://github.com/SoulaymaneK/daskeyboard-applet-github-action-status/issues",
+  "homePageUrl": "https://github.com/SoulaymaneK/daskeyboard-applet-github-action-status",
+  "developerRepoUrl": "https://github.com/SoulaymaneK/daskeyboard-applet-github-action-status",
   "license": "MIT",
+  "changelogUrl": "CHANGELOG.md",
+  "readMeUrl": "README.md",
+  "readMeEndUserUrl": "README_ENDUSER.md",
+  "engines": {
+    "das-keyboard-q": "3.2.3"
```

Comment:
Use the version of the Das Keyboard Q software that you used to test your applet .

You can find it on the bottom on the home page of the app:

![image](https://github.com/daskeyboard/daskeyboard-applet--action-status-for-github/assets/20217427/2b8d66a8-0fd8-4013-bc91-4bac02d9f93d)